src/components/paymentForm/PaymentForm.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import './PaymentForm.scss';
+import { MDBContainer, MDBRow, MDBCol, MDBIcon, MDBInput } from 'mdbreact';
+
+export default function PaymentForm() {
+  const paymentForm = {
+    paymentMethod: 'Payment Method',
+    creditCard: 'Credit Card',
+    paypal: 'Paypal',
+    name: 'Name',
+    cardNumber: 'Card Number',
+    MMYYYY: 'MM/YYYY',
+    CVC: 'CVC',
+    purchase: 'Purchase',
+  };
```

Comment:
now if you could move this to the __i18next__ and used it here, it will be perfect and ready to merge